products/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View
 
-# Create your views here.
+from .models import Category,CategoryFood,Food,FoodImage
+class FoodlistView(View):
+    def get(self, request):
+        category_filter = request.GET.get('category')
```

Comment:
명확하게 `id`를 붙여주는게 가독성을 높여줄 것 같네요!
```suggestion
        category_id = request.GET.get('category_id')
        # 또는
        category_id = request.GET.get('categoryId')
```

Reply:
category_id = request.GET.get('categoryId')로 변경완료 했습니다 !